dmutils/authentication.py

```diff
@@ -0,0 +1,25 @@
+from werkzeug.exceptions import Unauthorized
+
+
```

Comment:
Does this just send back the Auth header they sent to us?

Reply:
Not quite - this is used in cases when no auth header was provided in the request, the response adding a `WWW-Authenticate` header describing what scheme we want (`Bearer` in most cases) and what "auth domain" this is for (arbitrary but we'll mostly be using `main` I think).